### PR TITLE
fix inline for null body method

### PR DESCRIPTION
**Pull request by Jim8y**
this pr fixes https://github.com/neo-project/neo-devpack-dotnet/issues/1209

### CODE REVIEW

**Comment by Hecate2** [comment]
```csharp
        [MethodImpl(MethodImplOptions.AggressiveInlining)]
        private static void ArrowInline(int a, int b) => CallMethodThatReturnsInt(a, b);
```

**Review by shargon — APPROVED** [review]

**Comment by shargon** [comment]
> ```cs
>         [MethodImpl(MethodImplOptions.AggressiveInlining)]
>         private static void ArrowInline(int a, int b) => CallMethodThatReturnsInt(a, b);
> ```
> 
> The compiler should drop the returned value in this case.

If it's private yes, if it's public... Maybe we can create a new issue for it

File: tests/Neo.Compiler.CSharp.UnitTests/UnitTest_Inline.cs [hunk 1]
```diff
@@ -35,5 +35,11 @@ public void Test_NestedInline()
             Assert.AreEqual(new BigInteger(3), Contract.TestInline("inline_nested"));
             AssertGasConsumed(1071930);
         }
+
+        [TestMethod]
+        public void Test_ArrowMethod()
+        {
+            Assert.AreEqual(new BigInteger(3), Contract.ArrowMethod());
```
Comment (shargon) [inline tests/Neo.Compiler.CSharp.UnitTests/UnitTest_Inline.cs:42]: artifacts not updated

**Comment by Jim8y** [comment]
> ```cs
>         [MethodImpl(MethodImplOptions.AggressiveInlining)]
>         private static void ArrowInline(int a, int b) => CallMethodThatReturnsInt(a, b);
> ```
> 
> The compiler should drop the returned value in this case.

Fixed.

File: tests/Neo.Compiler.CSharp.UnitTests/UnitTest_Inline.cs (hunk 1 above)
Comment (Jim8y) [reply tests/Neo.Compiler.CSharp.UnitTests/UnitTest_Inline.cs:42]: done

**Review by shargon — APPROVED** [review]

**Pull request merged**